### PR TITLE
VEN-621 | Order info on the offer card 

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Fetch order info on the offer card
- Implement editing modal for creating and deleting `orderLine`s

### Closes :no_good_woman:
**[VEN-621](https://helsinkisolutionoffice.atlassian.net/browse/VEN-621)**: FE - Offer card: replace hardcoded invoice data with GQL 

### Manual testing :construction_worker_man:
- Go to a single application page
- `TARJOUS` card should show the order info
- Click on `Muokkaa` to open an editing modal

## Screenshots :camera_flash:
image
image

### CODE REVIEW

File: src/features/applicationView/offerCard/editForm/EditForm.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+  return (
+    <Formik
+      initialValues={initialValues}
+      onSubmit={(values) => onSubmit(values)}
+      validateOnBlur={false}
+      validateOnChange={false}
+    >
+      {({ values, errors, handleChange, handleSubmit }) => {
+        type Product = { id: string; label: string; price: number };
```
Comment (TatuArvela) [inline src/features/applicationView/offerCard/editForm/EditForm.tsx:92]: Should this type be outside of the return block?

File: src/features/applicationView/offerCard/editForm/EditForm.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+          return [...acc, product];
+        }, []);
+
+        const ProductsItems = products?.map(({ id, label, price }, i) => (
+          <ListItem key={i} className={styles.listItem}>
+            <Checkbox id={id} name={id} checked={values[id]} onChange={handleChange} label={label} />
+            <span>{formatPrice(price, i18n.language)}</span>
+          </ListItem>
+        ));
```
Comment (TatuArvela) [inline src/features/applicationView/offerCard/editForm/EditForm.tsx:111]: `products` could be moved before the return, and `ProductItems` could be inlined below

File: src/features/applicationView/offerCard/editForm/EditForm.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/features/applicationView/offerCard/editForm/EditForm.tsx]: Moved it up

File: src/features/applicationView/offerCard/editForm/EditForm.tsx (hunk 2 above)
Comment (Nizar-Rahme) [reply src/features/applicationView/offerCard/editForm/EditForm.tsx]: Good point! 
Fixed.

**Pull request merged**